src/routes/cyber/cyber-portal/resources/features.ts

```diff
@@ -6,23 +6,40 @@ type ProductFeature = {
 export const starterFeatures: ProductFeature[] = [
   {
     text: 'Anzahl E-Mails: 12',
-    detail: 'Anzahl der Phising Mails pro Anwender und Jahr'
-  },
-  {
-    text: 'E-Mail-Support',
-    detail:
-      'Unsere Reaktionszeit wird an Werktagen innerhalb 48 Stunden in deutscher Sprache gewährleistet.'
-  },
-  { text: 'Frequenz: 1 pro Monat' },
-  {
+    detail: 'Anzahl der Phising-Simulationen pro Empfänger und Jahr.'
+  },{ 
+    text: 'Frequenz: 30 Tage',
+    detail: 'Pro Monat erhalten Ihre Anwender eine Phishing-Simulation.'
+  },{ 
+    text: 'Versandzeit: Gleichzeitig',
+    detail: 'Alle Phishing-Simulationen gehen zur gleichen Zeit an die Anwender raus.'
+  },{
     text: 'Sprachniveau: Mittel',
-    detail:
-      'Das Sprachniveau variiert zw. niedrig und hoch. Bei mittlerem Niveau wird korrekte Rechtschreibung und Grammatik benutzt. Auf hohem Sprachniveau wird zusätzlich auf eine klare Semantik und Plausiblität geachtet.'
-  },
-  {
+    detail: 'Das Sprachniveau variiert zw. niedrig und hoch. Bei mittlerem Niveau wird korrekte Rechtschreibung und Grammatik benutzt. Auf hohem Sprachniveau wird zusätzlich auf eine klare Semantik und Plausiblität geachtet.'
+  },{
+    text: 'Absender: Authentisch',
+    detail: 'Authentische Absenderadressen sind z.B. Domains wie bundesgerichtshof-karlsruhe.de oder bund-und-laender.online, wohingegen eine Domäne wie why-how-what.info weniger echt wirkt.'
+  },{
+    text: 'Anrede: Personalisiert',
+    detail: 'In den Phishing-Simulationen wird der Name des Empfänger in der Begrüßung verwendet, also z.B. Sehr geehrter Herr Tim Kater.'
```

Comment:
typo: Empfängers

evtl. "Anrede" statt "Begrüßung"?
evtl. Beispiel in Anführungszeichen